### ISSUE TITLE
Create a site map

### ISSUE THREAD

**Issue by paidforby** [sudomesh/buildyourowninter.net#1]
relating to [bug 10](https://github.com/sudomesh/bugs/issues/10), which generated this repo. Can we create a site map? What do you want to see on buildyourowninter.net? It's a blank page, use your imagination.

**Comment by sierkje** [comment]
How about modeling it on how we've done our BYOI workshops? Maybe think of topics as the workstations?

**Comment by paidforby** [comment]
Yeah, cool idea! It can also be a home for stuff like [tunneldigger-lab](https://github.com/sudomesh/tunneldigger-lab), [operator manual](https://github.com/sudomesh/sudowrt-firmware/blob/master/operator_manual.md), [services guide](https://github.com/sudomesh/babeld-lab/blob/master/services_guide.md), and the heavily reworked [node flashing walkthrough](https://sudoroom.org/wiki/Mesh/WalkThrough). which are currently scattered across three repos and two websites.

**Comment by paidforby** [comment]
I created a sudopad for the site map and other notes: https://peoplesopen.net/pad/p/byoi_website
